hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/security/acl/OzoneObj.java

```diff
@@ -95,6 +95,7 @@ public StoreType getStoreType() {
     VOLUME(OzoneConsts.VOLUME),
     BUCKET(OzoneConsts.BUCKET),
     KEY(OzoneConsts.KEY),
+    OPEN_KEY(OzoneConsts.OPEN_KEY),
```

Comment:
Can you add some comment on OPEN_KEY is needed as ozone object type? 
Do we have the corresponding acl type semantics documented somewhere?